docs/application/dotnet/api/overview.md

```diff
@@ -28,25 +28,25 @@ You can efficiently add Tizen wearable device specific user interfaces. The exte
 
 ## TizenFX API
 
-The Tizen Platform-Specific API allows applications to call into platform-specific functionality from the shared code. This functionality enables Xamarin.Forms applications to do things a native application can do, without causing the portable part of the application to become littered with operating system-specific details.
+The Tizen platform-specific API allows applications to call into platform-specific functionality from the shared code. This functionality enables Xamarin.Forms applications to do things a native application can do, without causing the portable part of the application to become littered with operating system-specific details.
```

Comment:
We used to "Tizen Platform-Specific API" before. However, The name of API was change to "TizenFX API".
Please use "The TizenFX API" instead of "The Tizen platform-specific API".